cli/lib/kontena/cli/stacks/common.rb

```diff
@@ -94,9 +94,24 @@ module StackValuesFromOption
       def self.included(where)
         where.prepend InstanceMethods
 
-        where.option '--values-from', '[FILE]', 'Read variable values from YAML' do |filename|
-          values_from_file.merge!(::YAML.safe_load(File.read(filename)))
-          true
+        where.option '--values-from', '[FILE|STACK_NAME]', 'Read variable values from a YAML file or an installed stack', multivalued: true do |filename|
+          if File.exist?(filename)
```

Comment:
`kontena stack install --values-from ../some/path test/redis.yaml` => `DEBUG [Request]: GET http://localhost:9292/v1/stacks/development/../some/path`

Having this option either take a YAML file or a stack name feels pretty dirty, and having the `File.exist?` check work this way around feels particularly awful, leaking garbage into  API requests:

`kontena stack install --values-from '../some/path?lolwut=true' test/redis.yaml` => `172.17.0.1 - - [15/Sep/2017:11:36:01 +0000] "GET /v1/stacks/development/../some/path?lolwut=true HTTP/1.1" 404 2 0.0743`

IMO have a separate `--values-from-stack` option, or at least flip the logic around to be like `if filename =~ /\A(?!-)(\w|-)+\z/ && !File.exist?(filename)`... then that just means that `--values-from redis` will fail if you e.g. happen to be in a path that has a directory named `redis` - not ideal.